gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -156,6 +181,12 @@ export default function NxFormLayoutExample() {
       <NxFormGroup label="Upload a File" sublabel={<>Foo<br/>Bar</>} isRequired>
         <NxFileUpload files={files} isRequired isPristine={isFilePristine} onChange={onFileChange} />
       </NxFormGroup>
+      <NxFormGroup label="State" isRequired>
```

Comment:
This form group is marked as required yet the combobox doesn't have required validation

Reply:
Updated here 7047f0760fdf092735b418b118ce8b2643a1d17c.